lib/lknvball_web/resolvers/event.ex

```diff
@@ -105,4 +104,19 @@ defmodule LknvballWeb.Resolvers.Event do
   def create_event(_, _, _) do
     {:error, "expected input for event creation"}
   end
+
+  ## UPDATE EVENT MUTATION
+
+  def update_event(_, %{input: %{id: id} = input}, _) do
+    Lknvball.Events.get_event!(id)
```

Comment:
i did this in the node connection - but if you could leave a todo comment to actually handle failures instead of using the get_event! method (api will blow up if it isnt a valid id)

Reply:
Sounds good